docs/src/main/asciidoc/security-openid-connect-multitenancy.adoc

```diff
@@ -309,15 +321,15 @@ public class CustomTenantResolver implements TenantResolver {
 }
 ----
 
-In fact, this is how Quarkus OIDC resolves static custom tenants itself if no custom `TenantResolver` is registered.
+This is how Quarkus OIDC resolves static custom tenants if no custom `TenantResolver` is registered.
 
-A similar technique can be used with `TenantConfigResolver` where a `tenant-id` provided in the context can be used to return `OidcTenantConfig` already prepared with the previous request.
+A similar technique can be used with `TenantConfigResolver` where a `tenant-id` provided in the context can return `OidcTenantConfig` already prepared with the previous request.
 ====
 
 [NOTE]
 ====
 If you also use xref:hibernate-orm.adoc#multitenancy[Hibernate ORM multitenancy] or xref:mongodb-panache.adoc#multitenancy[MongoDB with Panache multitenancy] and both tenant IDs are the same
-and must be extracted from the Vert.x `RoutingContext` you can pass the tenant id from the OIDC Tenant Resolver to the Hibernate ORM Tenant Resolver or MongoDB with Panache Mongo Database Resolver
+and must be extracted from the Vert.x `RoutingContext`, you can pass the tenant ID from the OIDC Tenant Resolver to the Hibernate ORM Tenant Resolver or MongoDB with Panache Mongo Database Resolver
```

Comment:
```suggestion
and must be extracted from the Vert.x `RoutingContext`, you can pass the tenant id from the OIDC Tenant Resolver to the Hibernate ORM Tenant Resolver or MongoDB with Panache Mongo Database Resolver
```

As discussed earlier, lets keep capital ID to `ID token`

Reply:
Thanks. Fixed multiple instances.